tests/optimal_interpolation/test_integration.py

```diff
@@ -6,10 +6,32 @@
 import xhydro.optimal_interpolation.compare_result as cr
 import xhydro.optimal_interpolation.cross_validation as cv
 from xhydro.optimal_interpolation.functions.testdata import get_file
-
+import pooch
+from pathlib import Path
+from zupfile import ZipFile
```

Comment:
@richardarsenault 

PVI, there's built-in support for unzipping files in `pooch`: https://www.fatiando.org/pooch/latest/api/generated/pooch.Unzip.html.

Also, `zupfile` doesn't look quite right...

Reply:
indeed! Couldn't test and fix typos until I had access to the testdata repo to add the updated files etc. This was just a temporary push so I could work on other stuff in the meantime.

Will do and correct this. I used the instructions from the xhydro-testdata on how to process zipped files, so we might want to update that there too if we want to standardize things.